game/locations.rpy

```diff
@@ -90,7 +100,7 @@ init -20 python:
             self.on_entry = on_entry
             self.on_exit = on_exit
 
-        def getCurrentRoomImage(self):
+        def get_current_room_image(self):
```

Comment:
At first I thought you're renaming everything to follow `camelCase`, but here you're renaming to use `snake_case`. You also added some functions in `snake_case`. Technically Python pep8 suggests to use `snake_case` for functions, so I don't mind this change, but it's weird to see something being changed into `camelCase` and something into `snake_case`.